src/common/timeUtils.test.js

```diff
@@ -0,0 +1,26 @@
+import {
+  getDateString,
+  getUTCTimeString,
+  getHoursAndMinutes,
+  getDelayString,
+} from './timeUtils';
+
+describe('timeUtils', () => {
+  test('getDateString should be correct.', () => {
+    expect(getDateString(new Date(2020, 5, 30))).toBe('20200630');
+  });
+
+  test('getUTCTimeString should be correct.', () => {
+    expect(getUTCTimeString(new Date(2020, 5, 30, 11, 5, 1, 123))).toBe(
+      '9:5:1.123',
```

Comment:
In the function description of `getUTCTimeString` zero padding is used. @oterral What's excepcted?

Reply:
no idea